syntax = "proto3";

// 网关机的rpc server定义
// cd gateway/rpc 下执行protoc去编译
service Gateway {
  rpc DelConn (GatewayRequest) returns (GatewayResponse);
  rpc Push (GatewayRequest) returns (GatewayResponse);
}

message GatewayRequest{
  uint64 connID = 1;
  bytes data = 2;
}

message GatewayResponse {
  int32 code = 1;
  string msg = 2;
}